show_notes/Episode_11.md

```diff
@@ -0,0 +1,37 @@
+# Episode 11: Migration to RxJava 2 with Artur Dryomov from Juno
+
+### [How to listen & subscribe](https://github.com/artem-zinnatullin/TheContext-Podcast)
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/71)
+
+We've talked to Artur about Juno's way to migrate their Android Riders App from RxJava 1 to RxJava 2. We have covered:
+
+ - Migration strategy at Juno
+ - What's different in RxJava 2.0?
+ - Null values in RxJava 2
+ - Flowable or Observable?
+ - Other Rx Types: Single, Maybe and Completable
+ - RxJava 2 interop library
+ - RxJava 2: Default UncaughtExceptionHandler logs exceptions silently in jvm unit tests
+ - RxJava 3
+
+#### Guest:
+
+ - Artur Dryomov [GitHub](https://github.com/ming13)
+
+#### Hosts:
+
+  - Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+  - Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](http://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+
+#### Additional links:
+
```

Comment:
`[Juno's Android Rider Team RxJava 2 Migration Specification](https://github.com/gojuno/engineering/blob/master/specs/rxjava-2-migration.md)` 

It should be there at the time of publishing the episode, for now it's in the PR https://github.com/gojuno/engineering/pull/1